cio/src/rfds.rs

```diff
@@ -1492,6 +1476,14 @@ impl PDFStorage for RFDBranch {
     }
 }
 
+fn clean_quotes(content: &str) -> String {
```

Comment:
Why do we need to replace these quotes? I know sometimes they get used accidentally in Word, etc. but I would think that if someone uses them in asciidoc source, they probably meant to. Up to you, but I'd just drop it along with the rest of the de-unicodification.

Reply:
Only in that they are currently replaced and I didn't spend enough time to verify that they should be left as is. Ideally we don't perform any replacements. Certainly after merging this I'll take a better look and see if that replacement can be removed as well.